website/static/js/wikiMenu.js

```diff
@@ -83,7 +83,7 @@ function WikiMenu(data, wikiID, canEdit) {
                 columns.push({
                     folderIcons: true,
                     custom: function() {
-                        return m('a.fg-file-links', {href: item.data.page.url}, item.data.page.name);
+                        return m('a.fg-file-links', {href: item.data.page.url}, m.trust(item.data.page.name));
```

Comment:
I'm not sure this is a good solution. My concern is that it would open us up to a js/html injection attack. Can we do something a bit less extreme, like replacing the `&amp;`, `&gt;`, and `&lt;` in the `item.data.page.name` to `&`, `>`, and `<` respectively?